osdep/mac/swift_extensions.swift

```diff
@@ -52,6 +52,16 @@ extension NSEvent.ModifierFlags {
     public static let optionRight: NSEvent.ModifierFlags = .init(rawValue: UInt(NX_DEVICERALTKEYMASK))
 }
 
+extension String {
```

Comment:
If you are already doing all this dance, why not use pointer like before?

Also to you might want to make 
``` C
static const char *const swift_mpv_version = mpv_version;
```
to make those pointers more strict. Maybe this helps?

Reply:
i wasn't aware you could make the pointer const. thanks, that's a lot better.